ui/selectors/assets.ts

```diff
@@ -40,3 +62,153 @@ export function getAssetsMetadata(state: AssetsState) {
 export function getAssetsRates(state: AssetsRatesState) {
   return state.metamask.conversionRates;
 }
+
+const NON_EVM_TOKEN_NAMESPACE = 'token';
```

Comment:
`token` is a solana only namespace, to find if an asset is native or not we should look for `SLIP-44` as in the previous requested change of this comment: https://github.com/MetaMask/metamask-extension/pull/30389#discussion_r1962204374.

Reply:
fixed [5d71931](https://github.com/MetaMask/metamask-extension/pull/30389/commits/5d71931f5168b8430e3236866811b65dc53b79d8), thnx!

Reply:
I can make this change 👍

Reply:
https://github.com/MetaMask/metamask-extension/pull/30389/commits/5d71931f5168b8430e3236866811b65dc53b79d8